index.bs

```diff
@@ -561,12 +583,18 @@ random delay to deliver an [=aggregatable report=].
 
 <h3 id="parsing-filter-data">Parsing filter data</h3>
 
-To <dfn>parse filter data</dfn> given a |value| and a [=string=]
+A <dfn>registration type</dfn> is one of the following:
+
+<ul dfn-for="registration type">
+<li>"<dfn><code>source</code></dfn>"
+<li>"<dfn><code>trigger</code></dfn>"
+</ul>
+
+To <dfn>parse filter data</dfn> given a |value| and a [=registration type=]
 <dfn for="parse filter data"><var>registrationType</var></dfn>:
```

Comment:
Do we still need to define this var with the explicit type?

Reply:
Did we ever need to?

Reply:
Probably we can get rid of that?